code/game/objects/structures/tables_racks.dm

```diff
@@ -331,6 +331,13 @@
 	material_flags = MATERIAL_EFFECTS | MATERIAL_ADD_PREFIX | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
 	buildstack = null //No buildstack, so generate from mat datums
 
+/obj/structure/table/greyscale/set_custom_materials(list/materials, multiplier)
+	. = ..()
+	var/list/materials_list = list()
+	for(var/datum/material/current_material as anything in materials)
+		materials_list += "[current_material.name]"
+	desc = "A square [(materials_list.len > 1) ? "amagamation" : "peice"] of [english_list(materials_list)] on four legs. It can not move."
```

Comment:
```suggestion
	desc = "A square [(materials_list.len > 1) ? "amalgamation" : "piece"] of [english_list(materials_list)] on four legs. It can not move."
```